lib/internal/benchmark/report.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const {
+  NumberPrototypeToFixed,
+  globalThis,
+} = primordials;
+const { Intl } = globalThis;
+
+const { timer } = require('internal/benchmark/clock');
+
+const formatter = Intl.NumberFormat(undefined, {
+  notation: 'standard',
+  maximumFractionDigits: 2,
+});
+
+function reportConsoleBench(bench, result) {
+  const opsSecReported = result.opsSec < 100 ?
+    NumberPrototypeToFixed(result.opsSec, 2) :
+    NumberPrototypeToFixed(result.opsSec, 0);
+
+  process.stdout.write(bench.name);
+  process.stdout.write(' x ');
+  process.stdout.write(`${formatter.format(opsSecReported)} ops/sec +/- `);
+  process.stdout.write(formatter.format(
```

Comment:
Multiple writes like this are going to be fairly expensive on their own. Can this be coalesced? Or perhaps can this use `cork()/`uncork()` to improving buffering and reduce the number of individual flushing writes?

Reply:
We can build the whole string and then just `.write`, the calls were divided in this way just for readability purposes.